src/components/services-view/services-view.html

```diff
@@ -1,8 +1,17 @@
 
 <div class="services-view">
 
-  <overlay-panel show-panel="$ctrl.orderingPanelvisible" show-close="true" handle-close="$ctrl.closeOrderingPanel">
-    <order-service service-class="$ctrl.serviceToOrder" handle-close="$ctrl.closeOrderingPanel"></order-service>
+  <overlay-panel show-panel="$ctrl.orderingPanelVisible" show-close="true" handle-close="$ctrl.closeOrderingPanel">
+    <order-service
+        ng-if="$ctrl.selectedServiceClass"
+        service-class="$ctrl.selectedServiceClass"
+        handle-close="$ctrl.closeOrderingPanel">
+    </order-service>
+    <builder-dialog
```

Comment:
I'd argue as well that `order-service` tells you what it does, but not what it is. It doesn't tell you it's a dialog or wizard. FWIW, I had trouble finding the dialog code at first because of naming. But I agree we should be consistent between the two.

If we keep the verb pattern, I'd much rather call the `order-service` dialog `provision-service-instance` so that it matches the API. It's longer, but much more obvious if you know the service catalog API.

> `create-builder`, `new-builder`

You're not creating a builder. The builder already exists.

> `configure-builder`

Not exactly right either. You're creating a number of resources that support your application, using a builder image for your build config.

I thought about `create-app` which matches the command-line `oc new-app`, but `oc new-app` does other things as well.